data_steward/cdr_cleaner/clean_cdr_engine.py

```diff
@@ -14,6 +14,21 @@
 LOGGER = logging.getLogger(__name__)
 FILENAME = '/tmp/cleaner.log'
 
+FAILURE_MESSAGE_TEMPLATE = '''
```

Comment:
nit:  consider creating a constants module for this and other globals and moving it there.

Reply:
@lrwb-aou just wondering, what is the best practice for maintaining the logging related globals? Shoud they be moved to the constant module as well?

Reply:
I don't know.  Use your best judgement.  Eventually, I hope all this logging setup will be cleared out of this module and done in its own system wide logging setup module.  I _think_ the logging function doc string says something about needing to be properly placed in a separate module.

Reply:
@lrwb-aou  thanks for leaving this comment. I would move `FILENAME='/tmp/cleaner.log'` to the constants module and leave `LOGGER = logging.getLogger(__name__)` in `clean_cdr_engine.py` because it will be replaced by the system wide logging setup module eventually.

Reply:
Sorry I wasn't clear earlier.  Yes, you should leave `LOGGER = logging.getLogger(__name__)` here.  It will always have to be here if you want your logs to print from this module with the correct module name.  The other globals may be moved to a constants file.